.dockerignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
better to exclude only certain folders:

node_modules
build

Reply:
I will use .gitignore as a template, but I will also include the .env file.